src/lib/webpack/webpack-base-config.js

```diff
@@ -97,6 +100,37 @@ export default (env) => {
 			}
 		}),
 
+		// TypeScript
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
+									useCache: true,
+									configFileName: tsconfig,
```

Comment:
@marvinhagemeister `awesome-typescript-loader` is still much faster, although you can reach similar speeds with `ts-loader` with the [fork checker plugin](https://github.com/TypeStrong/ts-loader#faster-builds). I'm going to switch back to ATL, because all this stuff (including babel support) is "first-class."